pyaerocom/tools/aeroval_parallelise.py

```diff
@@ -95,40 +104,43 @@ def prep_files(options):
     return runfiles
 
 
-def run_queue(
-    runfiles,
-    qsub_host=QSUB_HOST,
-    qsub_cmd=QSUB_NAME,
-    qsub_dir=QSUB_DIR,
-    qsub_user=QSUB_USER,
-    qsub_queue=QSUB_QUEUE_NAME,
+def get_runfile_str_arr(
+    file,
+    queue_name=QSUB_QUEUE_NAME,
+    script_name=None,
+    wd=QSUB_DIR,
+    mail=f"{QSUB_USER}@met.no",
+    logdir=QSUB_LOG_DIR,
+    date=START_TIME,
 ):
-    """submit runfiles to the remote cluster"""
-
-    # to enable test usage, import fabric only here
-    from fabric import Connection
-    import subprocess
-
+    """create list of strings with runfile for gridengine"""
     # create runfile
     runfile_arr = []
     runfile_arr.append("#!/bin/bash -l")
     runfile_arr.append("#$ -S /bin/bash")
-    runfile_arr.append("#$ -N AEROVAL_NAME")
-    runfile_arr.append("#$ -q research-el7.q")
+    # runfile_arr.append("#$ -N AEROVAL_NAME")
+    runfile_arr.append(f"#$ -N {PurePath(file).stem}")
+    # runfile_arr.append("#$ -q research-el7.q")
+    runfile_arr.append(f"#$ -q {queue_name}")
     runfile_arr.append("#$ -pe shmem-1 1")
-    runfile_arr.append("#$ -wd /home/UUSER/data/aeroval-local-web/pyaerocom_config/config_files")
+    # runfile_arr.append("#$ -wd /home/UUSER/data/aeroval-local-web/pyaerocom_config/config_files")
+    runfile_arr.append(f"#$ -wd {PurePath(file).parent}")
     runfile_arr.append("#$ -l h_rt=96:00:00")
     runfile_arr.append("#$ -l s_rt=96:00:00")
-    runfile_arr.append("#$ -M UUSER@met.no")
+    # runfile_arr.append("#$ -M UUSER@met.no")
+    if mail is not None:
+        runfile_arr.append(f"#$ -M {mail}")
     runfile_arr.append("#$ -m abe")
     runfile_arr.append("#$ -l h_vmem=64G")
     runfile_arr.append("#$ -shell y")
     runfile_arr.append("#$ -j y")
-    runfile_arr.append("#$ -o /lustre/storeA/project/aerocom/logs/aeroval_logs/")
-    runfile_arr.append("#$ -e /lustre/storeA/project/aerocom/logs/aeroval_logs/")
+    # runfile_arr.append("#$ -o /lustre/storeA/project/aerocom/logs/aeroval_logs/")
+    # runfile_arr.append("#$ -e /lustre/storeA/project/aerocom/logs/aeroval_logs/")
+    runfile_arr.append(f"#$ -o {logdir}/")
+    runfile_arr.append(f"#$ -e {logdir}/")
 
-    runfile_arr.append('logdir="/lustre/storeA/project/aerocom/logs/aeroval_logs"')
-    runfile_arr.append("date=$(date '+%Y%m%d_%H%M%S')")
+    runfile_arr.append(f"logdir='{logdir}/'")
+    runfile_arr.append(f"date={date}")
```

Comment:
I did not know textwrap. Thanks for the suggestion. The other checks are not that relevant yet since that really was WIP. I had just added some potentially needed stuff.

You are looking at my draft PRs a bit early :-)